test/unit_tests/modules/util/web-storage-proxy.js

```diff
@@ -0,0 +1,262 @@
+'use strict';
+
+var chai = require( 'chai' );
+var sinon = require( 'sinon' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+
+describe( 'web-storage-proxy', function() {
+  var webStorageProxy, sandbox, setItem, getItem, removeItem, setStorage;
+
+  jsdom();
+
+  before( function() {
+    webStorageProxy =
+      require( '../../../../src/static/js/modules/util/web-storage-proxy.js' );
```

Comment:
Mostly mention this though because of http://eslint.org/docs/rules/global-require.html